cerealize/test/impl.cc

```diff
@@ -93,7 +96,7 @@ TEST(Cerealize, NamedJSON) {
 };
 
 TEST(Cerealize, BinarySerializesAndParses) {
-  FileSystem::CreateDirectory(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirectoryParameters::Silent);
+  FileSystem::CreateDir(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirParameters::Silent);
```

Comment:
FileSystem::mkdir ? :)

> On Feb 9, 2015, at 20:51, John Babak <notifications@github.com> wrote:
> 
> In cerealize/test/impl.cc <https://github.com/KnowSheet/Bricks/pull/61#discussion_r24359144>:
> 
> > @@ -93,7 +96,7 @@ TEST(Cerealize, NamedJSON) {
> >  };
> >  
> >  TEST(Cerealize, BinarySerializesAndParses) {
> > -  FileSystem::CreateDirectory(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirectoryParameters::Silent);
> > +  FileSystem::CreateDir(FLAGS_cerealize_test_tmpdir, FileSystem::CreateDirParameters::Silent);
> Maybe add a comment next to CreateDir so that someone won't rename it back?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/KnowSheet/Bricks/pull/61/files#r24359144>.
>